css/properties/initial-letter.json

```diff
@@ -57,14 +57,14 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": false
+                "version_added": "9"
               },
               "safari_ios": "mirror",
               "samsunginternet_android": "mirror",
               "webview_android": "mirror"
             },
             "status": {
-              "experimental": true,
+              "experimental": false,
```

Comment:
Form end users' perspective, how can a property be experimental but its value is non-experimental?

Reply:
That's a fair point.  We'll have to update the experimental status linter to listen for the parent's experimental status.  The linter just sees that this specific feature is supported in two browser engines without partial implementation or prefixes/alternative names, so it says that the feature is non-experimental.

This would require a linter infrastructure overhaul, though.  I feel that we should merge this PR as-is and resolve the linter issues at a later date.

Reply:
I think we should mark both the value and the parent property as `experimental: false `and move on.